src/App.tsx

```diff
@@ -1,26 +1,198 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useState } from 'react';
+import * as todosService from './api/todos';
+import { Todo } from './types/Todo';
+import { Header } from './component/Header/Header';
+import { TodoList } from './component/TodoList/TodoList';
+import { Footer } from './component/Footer/Footer';
+import { TodoItem } from './component/TodoItem/TodoItem';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState('');
+  const [status, setStatus] = useState('all');
+  const [todo, setTodo] = useState<Todo>();
+  const [todosIsLoading, setTodosIsLoading] = useState<number[]>([]);
+  const [isInputDisabled, setInputDisabled] = useState(false);
+
+  useEffect(() => {
+    todosService
+      .getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setError('Unable to load todos');
+      })
+      .finally(() => {});
+  }, [todosIsLoading, todos]);
```

Comment:
```suggestion
  }, [todosIsLoading]);
```
please, remove that to pass the tests